compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -176,6 +176,11 @@ impl<'hir> Map<'hir> {
         self.tcx.local_def_id_to_hir_id(def_id)
     }
 
+    #[inline]
+    pub fn opt_local_def_id_to_hir_id(self, def_id: LocalDefId) -> Option<HirId> {
```

Comment:
Yes, I did this to leave it similar to what we do for `local_def_id_to_hir_id` just above. I'm fine with both things, if you prefer I can inline it manually, no problem.

Reply:
I prefer just calling `self.tcx.opt_local_def_id_to_hir_id` but maybe someone else doesn't.  Having this function might lead someone to call `tcx.hir().opt_local_def_id_to_hir_id(..)` instead of just `tcx.opt_local_def_id_to_hir_id(..)`, or may be confused between which of them to call.

Reply:
Agreed and changed it, although I think I'd do the same then for `local_def_id_to_hir_id`.